src/rendering/SceneManager.cc

```diff
@@ -74,6 +78,10 @@ class ignition::gazebo::SceneManagerPrivate
   /// \brief Map of light entity in Gazebo to light pointers.
   public: std::map<Entity, rendering::LightPtr> lights;
 
+  /// \brief Map of particle emitter entity in Gazebo to particle emitter
+  /// pointers.
```

Comment:
```suggestion
  /// rendering pointers.
```

Reply:
Done in 2e28aae0d51a9abe35018e891f6c87731e24a76f